### ISSUE TITLE
Interference with some unknown add-on (see list)

### ISSUE THREAD

**Issue by galantra** [glutanimate/popup-dictionary#20]
#### Problem description
#12 
See [here](https://github.com/glutanimate/popup-dictionary/issues/12#issuecomment-521127159).

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [X] I've restarted Anki to see if it helps
 - [X] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [X] I've verified that I use the latest version of Anki by checking at https://apps.ankiweb.net#download
 - [ ] I've tried to disable other add-ons to see if there are any interactions present
 - [X] My issue disappears when I hold shift while starting Anki.
 - [X] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.

*Please open Anki, go to Help → About, click on "Copy Debug Info", and paste the result between the backticks below (if the button does not appear you are using an older version of Anki 2.1 and will need to update first):*

```
Anki 2.1.15 (6661ba8a)
Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
```
*Please fill in details about your operating system (Windows/macOS/Linux, which version):*

- OS: Windows
- version: 10

*Please open Anki, go to Tools → Add-ons, take a screenshot of your installed add-ons, and paste it below:*

Edit Field During Review
TagSelectorV2 (disabled)
Pennywise Add Only Once Option
extended html editor
Speed Focus Mode auto-alert auto-reveal auto-fail
ReMemorize Ex Rated0Search
Color Confirmation
Search in formatted text in any fields
Straight Reward Ease Factor grows at 5 straight Good rating in review
Pretzel Logic Retention Benchmarking
Convert Subdecks to Tag Hierarchy
HoochiePapa Randomize New Queue
ImageResizer
Add Table
Change cards decks prefix (disabled)
Correcting a bug in anki which makes new card appearing in wrong order
Insert Randomized Lists multiple choice
Cacher In The Rye Windows
Quick note and deck buttons Fork for 21
CTRLF5 to Refresh the Browser
Image Occlusion Enhanced for Anki 21 alpha
A half-automatic Forvo Downloader
HoochieMama Randomize Rev Queue
Symbols
Another Retreat go upstream the history for the next interval at lapse (disabled)
Night Mode
Cookie monster (disabled)
Cardistry Dynamically Adjust New Cards
Pop-up Dictionary beta
Reviewer to Browser choosing what to display (disabled)
Copy notes
Full Screen Toggle
Touch Screen - draw and write on your cards (disabled)
Refocus Card when Reviewing 21
MIA Retirement Addon (disabled)
More consistent cards generation
Agent Orange Pseudo-Fuzz Defoliator
Hoarders Delight RecycleTrash Bin
Search in Add Card Dialog
CrowdAnki JSON exportimport Edit history Collaborate on deck creation
Push Paradox Sibling Cardistry
Fast Word Query Multi-threaded queries for words from local or web dictionaries
Interactive Cards for language learning
Browser MaximizeHide TableEditor
Stats Overview Pie Graph with Distinct Learning and-or Relearning Sections (disabled)
Large and Colorful Buttons
Anki Excel Sync
Quick  Easy
Zoom 21
HoochieBaby Queue Controller  Day-Lrn Q
Feed to Anki for Word of the Day RSS
Ignore accents in browser search
Fill the blanks - Multiple typecloze support
Export cards selected in the Browser
Customize Sidebar
If a note has no more card warns instead of deleting it
Add a tag to notes with missing media
AnkiConnect
Enhanced cloze 21 (disabled)
Load Balanced Scheduler
Le Petit Debugger (disabled)
Automatic Basic to Cloze (disabled)
AutoDefine - Automatically define vocabulary words with pronunciations and image
Card Info Bar for Browser
Highlight Search Results in the Browser
Maximum image height in card editor
Customize Keyboard Shortcuts
Empty cards returns more usable informations
Progress Graphs and Stats for Learned and Matured Cards (disabled)
Batch Editing
Mini Format Pack
Quick tagging 21
Cross Profile Search and Import
Add Hyperlink
ReMemorize Rescheduler with sibling and logging v114
Opening the same window multiple time
Holdem Cardfield Preserves add card history
Clickable Tags v10 (disabled)
HTML Editor Tweaks v01
Bury related notes
Keep model of add cards
Allows empty first field during adding and import
Ease Factor Histogram
Replay buttons on card
Improving change note type
Frozen Fields
Long term backups
Improved Quizlet to Anki 21 Importer
add-on window search bar
Web Query  (disabled)
avgEase
Numeric Keypad Remapping  ease bugs fixed (disabled)
Syllabus - Anki Statistics and Export by Tag and Deck (disabled)
Clean the default deck (disabled)
Life Drain
Polar Connect - PDF and Incremental Reading with Anki Sync (disabled)
More Overview Stats 21 (disabled)
copy note contents to new Add window from Browser Reviewer Add EditCurrent
3 add-ons merged quicker anki explain deletion explain database check
Increase autosave frequency
Anki IPA
extended table editor
AwesomeTTS Google Cloud Text-to-Speech unofficial
Echelon - Improved Hierarchical Tags for Anki 21 (disabled)
Create Filtered Deck from the Browser
Reflect On Mee - Helps Alleviate RGS (disabled)
Open Added Today from Reviewer
Web Browser - Search selection Import texts and images automatically
Advanced Browser
Enhance main window
Slackers Delight PostponeDefer Button
Knockout Deck Subset Learning
MorphMan for Anki 21 (disabled)
Undelete deleted note
True Retention by Card Maturity
Editor Autocomplete for Anki 21
Incremental Reading v4103
Text Rendering Fix (disabled)
Correct Answers Graph
No Penalties or Boosting (disabled)
PassFail (EN) (disabled)
ResetEZ (disabled)
bettertags
Cloze Overlapper
more_answer_buttons (disabled)
pez_dispenser (disabled)
rememorize_btn
Review Heatmap
roommate_switch
show_warning_if_no_tags_when_adding_cards
Spell Checker
touch_wood
visual_feedback
wordsworth

**Comment by glutanimate** [comment]
Wow, that's a lot of add-ons! OK, so going through each of them one by one would take an eternity. How about this, though: You could start by disabling half of them at once. ( Go to Tools → Add-ons, shift-click to select half of the list, then use the "Toggle Enabled" button). Then, once you've figured out which half the culprit is in you could do the same thing for that half, continuing on until you've narrowed things down as much as possible.

**Comment by galantra** [comment]
This is weird. I went through all of the add-ons (not single, but by batch) and the only reponse I received was that pop-up dictionary may slow anki down because there are >1000 words (I am adumbrating the message here in my own words). I was double-clicking the word "because".
The add-ons I had set up differently from my usual settings:

![grafik](https://user-images.githubusercontent.com/52420923/63034762-28d8c480-beba-11e9-9783-6541fc55e2e8.png)

**Comment by randomman321** [comment]
I know which add on it is. It is "Edit Field During Review" : https://ankiweb.net/shared/info/385888438

I have the same issue, and when I uninstalled "Edit Field During Review", my pop up dictionary is working again. 

It would be great if both add ons would work with each other, but I'm not sure if it is a complex fix. 

Cheers!

**Comment by galantra** [comment]
I too have deleted both "edit during review" add-ons now.
Unfortunately, the dictionary still won't pop up.
The only response I can generate is this (when double-clicking on a very frequent word):
![grafik](https://user-images.githubusercontent.com/52420923/63743445-90204c80-c89c-11e9-9f36-8dabf801ee6e.png)
